client/src/Annotator/Info/useDataset.tsx

```diff
@@ -58,11 +64,29 @@ const useDataset: IUseDataset = imageId => {
         [enqueueSnackbar],
     );
 
+    const copyAnnotations = useCallback(
+        async (imageId: number, id: number, categoriesIds: number[]) => {
```

Comment:
I would say that function taking 3 args should use named args (pass the object into the function)